aws/resource_aws_accessanalyzer_analyzer_test.go

```diff
@@ -112,6 +112,38 @@ func testAccAWSAccessAnalyzerAnalyzer_Tags(t *testing.T) {
 	})
 }
 
+// This test can be run via the pattern: TestAccAWSAccessAnalyzer
+func testAccAWSAccessAnalyzerAnalyzer_Type_Organization(t *testing.T) {
+	var analyzer accessanalyzer.AnalyzerSummary
+
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_accessanalyzer_analyzer.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheckAWSAccessAnalyzer(t)
+			// testAccOrganizationsAccountPreCheck(t)
```

Comment:
should we consider keeping this check to skip this test instead of failing? 🤔

Reply:
Oi! Good catch. Extraneous code comment there.